NTumbleBit/ClassicTumbler/Client/PaymentStateMachine.cs

```diff
@@ -455,6 +455,14 @@ public CycleProgressInfo Update()
                                 Logs.Client.LogDebug("Still solving puzzles...");
                                 break;
                             }
+
+							int connectionCount = Services.BlockExplorerService.GetConnectionCount();
```

Comment:
It may be possible to move this slightly later in the payment phase without adverse effects, but I am not opposed to merging this as it stands